src/Donation/DonationStatusEnum.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace AppBundle\Donation;
+
+use MyCLabs\Enum\Enum;
```

Comment:
:puke:, why adding this lib to EM?

Reply:
Not me ^^, last time I did it in class, somebody told me to use Enum. I agree with you this can be done in Donation class directly.

Reply:
I did, this is a great lib. What would you do instead ?

Reply:
Note that you are not supposed to use it every single. Use it when it makes sense to you. I did not check for that case.